gtsam/geometry/Cal3Bundler.h

```diff
@@ -98,11 +98,23 @@ class GTSAM_EXPORT Cal3Bundler {
     return k2_;
   }
 
+  /// image center in x
+  inline double px() const {
+    return u0_;
+  }
+
+  /// image center in y
+  inline double py() const {
+    return v0_;
+  }
+
+  /// @deprecated
```

Comment:
No, we have this #ifdef thingy, remember ? :-)

Reply:
Uhhh I'm stupid